mojo/stdlib/src/utils/variant.mojo

```diff
@@ -390,3 +391,27 @@ struct Variant[*Ts: CollectionElement](
             if _type_is_eq[Ts[i], T]():
                 return i
         return Self._sentinel
+
+    @staticmethod
+    fn is_type_supported[T: CollectionElement]() -> Bool:
+        """Check if a type can be used by the `Variant`.
+
+        Example:
+        ```mojo
+        def MyFunction(mut arg: Variant):
```

Comment:
```suggestion
        def my_function(mut arg: Variant):
```

Reply:
I think that `my_function` does not provide any value in understanding the purpose of this method. Maybe we should pick more self explanatory name?